src/build.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
+    println!("cargo:rustc-link-search=native=/usr/local/opt/openssl/lib");
```

Comment:
As long as the linking is happening in the `-sys` module then we don't need any linker flags at this level.